### PR TITLE
Obtain full URL from config file

### PR DESCRIPTION
**Pull request by oliver-walker-hms**
No longer append the endpoint, have the user
specify the full URL.

### CODE REVIEW

**Comment by oliver-walker-hms** [comment]
Need to fix code style, please wait to review

**Comment by oliver-walker-hms** [comment]
Code style is fixed

File: config/ThingworxConnectorConfig.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
    "LogLevel": 1,
-   "ThingworxBaseUrl": "https://setme.biz",
+   "ThingworxBaseUrl": "https://setme.biz/Thingworx/Things/ConnectorHost/Services/TakeInfo",
```
Comment (TomKimsey) [inline config/ThingworxConnectorConfig.json:3]: Is this still considered the "base url" now that the endpoint is required?

**Review by TomKimsey — COMMENTED** [review]
do the endpoints change? it seems like its easier just to require the base url / ip address.

File: config/ThingworxConnectorConfig.json (hunk 1 above)
Comment (alexjhawk) [reply config/ThingworxConnectorConfig.json:3]: Good point - should be renamed

**Review by alexjhawk — CHANGES_REQUESTED** [review]

**Comment by alexjhawk** [comment]
> do the endpoints change? it seems like its easier just to require the base url / ip address.

Yes, they can change. The connector itself will be updated following these changes, and one update will be to include the capability for each device to have its own endpoint and app key.

**Comment by oliver-walker-hms** [comment]
I will add a commit renaming ThingworxBaseUrl

**Review by alexjhawk — APPROVED** [review]

**Review by TomKimsey — CHANGES_REQUESTED** [review]
These two commits should be combined. There shouldnt be a point where the config key doesnt match the content.

**Review by alexjhawk — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**